app/uat_actions/uat_actions/test_submission.rb

```diff
@@ -323,12 +348,8 @@ def default_purpose_name
 
   # Any helper methods
 
-  def study
-    UatActions::StaticRecords.study
-  end
-
   def project
-    UatActions::StaticRecords.project
+    Project.find_by(name: project_name) || UatActions::StaticRecords.project
```

Comment:
Similar to study, if project_name is explicitly specified, it needs a validation.

Reply:
added